### DISCUSSION TITLE
terraform installation

### DISCUSSION THREAD

**Discussion by vikram9491** [learndevopsonline/learndevopsonline#1336] [Q&A]
![Screenshot from 2023-08-15 15-01-31](https://github.com/learndevopsonline/learndevopsonline/assets/67426874/87a995e8-8ac8-4f31-9576-0b9e5ec0ae02)

hi @r-devops  terraform init is working while giving plan command getting some configuration issue , i installed terraform using labauto tool

**Comment by sreedharm07** [comment]
Hi @vikram9491 , attach IAM role (with admin permissions) to the workstation.

**Reply by vikram9491** [reply]
@sreedharm07  thanks for info

**Reply by vikram9491** [reply]
@sreedharm07  do u know in which day session that I AM permissions are added i couldn't find the video ...?

**Reply by sreedharm07** [reply]
day 21
